src/Mod/Path/PathScripts/PathFeatureExtensionsGui.py

```diff
@@ -299,13 +301,11 @@ def _initializeExtensions(self, obj):
             self.form.extensionEdit.setDisabled(True)
         self.setExtensions(self.extensions)
 
-    def updateQuantitySpinBoxes(self, index=None):
-        prevValue = self.form.defaultLength.text()
-        self.defaultLength.updateSpinBox()
-        postValue = self.form.defaultLength.text()
-
-        if postValue != prevValue:
-            PathLog.debug("updateQuantitySpinBoxes() post != prev value")
+    def _updateExtensionSizes(self, index=None):
+        """_updateExtensionSizes(index=None)...
+        Slot method to update extensions due to change in Default Length."""
+        if self.form.defaultLength.text() != self.lastDefaultLength:
+            self.lastDefaultLength = self.form.defaultLength.text()
```

Comment:
Are you sure you don't have to call `self.defaultLength.updateSpinBox()` anymore?

Reply:
Updates are being done by done by the event handler in the `QuantitySpinBox` class of the PathGui module.  Anytime the event is `FocusIn` it is calling it.  However, I was unsure on this one.  My tests show that values are being passed as needed for the extensions, and reloaded correctly when editing.  When I left it in, I was seeing double updates of the SpinBox, so I took it out.   I will place it back, just to be sure I am not messing something up in the background that I am not seeing.

Thanks for the review!

Reply:
Okay.  Just added back the `updateSpinBox()` call.